packages/@ember/object/lib/computed/reduce_computed_macros.ts

```diff
@@ -966,7 +966,7 @@ export function setDiff(setAProperty: string, setBProperty: string) {
       return emberA();
     }
     if (!isNativeOrEmberArray(setB)) {
-      return emberA(setA);
+      return setA;
```

Comment:
Huh. Are you sure?

Reply:
@chriskrycho and I talked elsewhere. This is the summary in case anyone else stumbles across this:

tl;dr emberA(setA) is a type error
I don't think you can correctly convert a non-js array EmberArray to a NativeArray
prior to my change, setDiff claims to return unknown[] | NativeArray<unknown>
after removing the emberA it becomes unknown[] | EmberArray<unknown> which I think is actually correct enough
if we really wanted a NativeArray we could do emberA(setA.toArray()) or equivalent
docs just say "computes a new array